docs/architecture.md

```diff
@@ -0,0 +1,63 @@
+# Architecture
+
+- Raw data comes in whatever form
+- They get cleaned up to some common schema
+- A "model" is everything between the common-schema input and some common-schema output
```

Comment:
Worth elaborating on our common output schema? Posterior samples of proportions per taxon-location-day for all days in a range?